setup.py

```diff
@@ -12,14 +12,13 @@
 import sys
 
 from setuptools import setup, Extension
-from Cython.Build import cythonize
 
 from comdb2 import __version__
 
 ccdb2 = Extension("comdb2._ccdb2",
                   extra_compile_args=['-std=c99'],
                   libraries=['cdb2api', 'protobuf-c'],
-                  sources=["comdb2/_ccdb2.pyx"])
+                  sources=["comdb2/_ccdb2.pyx", "comdb2/_cdb2api.pxd"])
```

Comment:
@mariocj89 Are you certain that including the `.pxd` is necessary?  The call to cythonize was introduced in de415a27e970ffc1caaa4db9f27f87fac3667398, and before that point the `.pxd` wasn't included in `sources`.

Reply:
Interesting, why was that added? 

There are multiple options here, adding it here ensures that the file is recognized as a source and included in the sdist, we can also just added to the MANIFEST

An option here as well is to include the .c in the sdist and cython wont be needed as well.